src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
   margin: 0;
```

Comment:
The universal selector (*) is setting a background color of #fff for all elements. This might override background colors for elements where a different background is desired. Consider applying this style more selectively if needed.